ibis/backends/pandas/tests/execution/test_timecontext.py

```diff
@@ -132,6 +134,44 @@ def test_context_adjustment_window(
     tm.assert_series_equal(result, expected)
 
 
+def test_trim_with_timecontext(time_df3):
+    """ Unit test `trim_with_timecontext` in Window execution
+    """
```

Comment:
Move this to the same line? (or is it too long)

Reply:
Seems `black` formatted this?

Reply:
kk